### ISSUE TITLE
accessibility review

### ISSUE THREAD

**Issue by BenSibley** [BenSibley/issues#38]

**Comment by BenSibley** [comment]
Header image should have alt text if image has alt text.
Header image link should be visible when selected.
Header image link should show in logical order.
New menus should have ARIA roles
New menus should be keyboard accessible
Featured sliders?
